WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -16,12 +18,18 @@ namespace WalletWasabi.Gui.Controls.WalletExplorer
 {
 	public class WalletViewModel : WalletViewModelBase
 	{
-		private ObservableCollection<WasabiDocumentTabViewModel> _actions;
+		private ObservableCollection<ViewModelBase> _actions;
```

Comment:
Please use a private property instead of fields to be consistent with the rest of the code.

Reply:
It is already, this is the backing field for the property!

```
public ObservableCollection<ViewModelBase> Actions
		{
			get => _actions;
			set => this.RaiseAndSetIfChanged(ref _actions, value);
		}
```

Reply:
ACK!